crates/turborepo-repository/src/change_mapper/mod.rs

```diff
@@ -183,13 +193,26 @@ impl<'a, PD: PackageChangeMapper> ChangeMapper<'a, PD> {
                     }
 
                     // We don't have the actual contents, so just invalidate everything
-                    Some(None) => {
-                        debug!("no previous lockfile available, assuming all packages changed");
+                    LockfileContents::UnknownChange => {
+                        // this can happen in a blobless checkout
+                        debug!(
+                            "we know the lockfile changed but we don't have the contents so we \
+                             have to assume all packages changed and rebuild everything"
+                        );
                         Ok(PackageChanges::All(
                             AllPackageChangeReason::LockfileChangedWithoutDetails,
                         ))
                     }
-                    None => Ok(PackageChanges::Some(changed_pkgs)),
+
+                    // We don't know if the lockfile changed or not, so we can't assume anything
+                    LockfileContents::Unknown => {
+                        debug!("no previous lockfile available, assuming all packages changed");
```

Comment:
The debug comment doesn't match the behavior. We aren't adding any additional changed packages in this branch.

Reply:
`LockfileContents::Unknown` was removed in bd24c7c623c058a67e1ba51e34182edecbd41d78